governator-test-junit/src/main/java/com/netflix/governator/guice/test/ReplaceWithMock.java

```diff
@@ -0,0 +1,58 @@
+package com.netflix.governator.guice.test;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import org.mockito.Answers;
+
+/**
+ * Creates an override binding for the annotated type. 
+ * 
+ * Example:
+ * <pre>
+ * {@literal @}RunWith(GovernatorJunit4ClassRunner.class)
+ * {@literal @}ModulesForTesting({ SomeTestModule.class })
```

Comment:
Do you need the ModuleForTesting annotation even when you are using ReplaceWithMock?